docs/get_started.md

```diff
@@ -28,51 +28,75 @@
-   </p>
-   -->
+@endsphinxdirective
+
+## <a name="quick-start-example"></a>1. Quick Start Example (No Installation Required)
+
+<img src="https://user-images.githubusercontent.com/15709723/127752390-f6aa371f-31b5-4846-84b9-18dd4f662406.gif">
+
+Try out OpenVINO's capabilities with this quick start example that estimates depth in a scene using an OpenVINO monodepth model. <a href="https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F201-vision-monodepth%2F201-vision-monodepth.ipynb">Run the example in a Jupyter Notebook inside your web browser</a> to quickly see how to load a model, prepare an image, inference the image, and display the result.
+
    
-   <h3><a name="get-started-tutorials">Tutorials, Samples, and Demo Applications</a></h3>
+## <a name="install-openvino"></a>2. Install OpenVINO
    
-   <p>After all the installation and configuration steps are done, you are ready to run your first inference and learn the workflow. Here is a set of hands-on demonstrations of various complexity levels to guide you through the process. You can run Jupyter notebooks, code samples, or demo applications.</p>
- 
-   <div id="GSG_nextstepchoice">
-     <a href="tutorials.html" >
-        <h4>Interactive Tutorials - Jupyter Notebooks (Python) </h4>
-        <p>Learn from a choice of interactive Python tutorials targeting typical OpenVINO use cases.</p>
-     </a> 		
-     <a href="openvino_docs_OV_UG_Samples_Overview.html" >
-        <h4>OpenVINO Samples	</h4>
-        <p>See ready-made applications explaining OpenVINO features and various use-cases.		</p>
-     </a> 
-     <a href="openvino_inference_engine_ie_bridges_python_sample_speech_sample_README.html" >
-        <h4>Reference Implementation For Speech Recognition Apps (Python)</h4>
-        <p>Use a speech recognition demo and Kaldi model conversion tool as reference. </p>
-     </a>
-     <a href="openvino_inference_engine_samples_speech_sample_README.html" >
-        <h4>Reference Implementation For Speech Recognition Apps (C++)</h4>
-        <p>Use a speech recognition demo and Kaldi model conversion tool as reference. </p>
-     </a>
-     <a href="omz_demos.html" >
-        <h4>Open Model Zoo Demo Applications</h4>
-        <p>See Open Model Zoo demo applications to use robust application templates for implementing specific deep learning scenarios. </p>
-     </a>
-     <a href="http://devcloud.intel.com/edge/" >
-        <h4>Intel® DevCloud 	</h4>
-        <p>Develop, test, and run your OpenVINO solution for free on a cluster of the latest Intel® hardware. </p>
-     </a> 
-   </div>
-   <div style="clear:both;"> </div>
-
-@endsphinxdirective
\ No newline at end of file
+Visit the [Install OpenVINO Overview page](./install_guides/installing-openvino-overview.md) to view options for installing OpenVINO and setting up a development environment on your device.
```

Comment:
```suggestion
See the [Installation Overview page](./install_guides/installing-openvino-overview.md) for options of installing OpenVINO and setting up a development environment on your device.
```

Reply:
How about this? Feel free to push back if you don't like it 🙂

"See the Installation Overview page for options to install OpenVINO and set up a development environment on your device."